lib/tasks/chf_tasks.rake

```diff
@@ -56,6 +56,31 @@ namespace :chf do
     puts "total: #{reporter.matches.size}"
   end
 
+  desc 'Reindex all GenericWorks'
+  task reindex_works: :environment do
+    # Like :reindex, but only GenericWorks, makes it faster,
+    # plus let's us use other solr techniques to make it faster,
+    # and allows us to add a progress bar easily.
+
+    add_batch_size = ENV['ADD_BATCH_SIZE'] || 50
+
+    progress_bar = ProgressBar.create(:total => GenericWork.count, format: "%t: |%B| %p%% %e")
+    solr_service_conn = ActiveFedora::SolrService.instance.conn
+    batch = []
+
+    GenericWork.find_each do |work|
+      batch << work.to_solr
+
+      if batch.count % add_batch_size == 0
```

Comment:
What happens to the last set of records if the total isn't a multiple of 50? Do we need an extra add outside of the loop to catch those?